packages/mix/lib/src/core/element.dart

```diff
@@ -25,12 +25,12 @@ typedef StyledAttribute = SpecAttribute;
 @Deprecated('Use Mixable instead')
 typedef Dto<Value> = Mixable<Value>;
 
-abstract class Mixable<Value> extends StyleElement {
+abstract class Mixable<Value> with EqualityMixin {
   const Mixable();
 
-  Value resolve(MixData mix);
+  Object get mergeKey => runtimeType;
```

Comment:
I think we can remove the merge key from mixable since the style element already has it.

Reply:
Done